Automattic/jetpack
# Boost: Update pricing component

Fixes https://github.com/Automattic/jetpack/issues/32251

## Proposed changes:

* Update the pricing component to match designs. Also update the Boost pricing section legend.
* Designs pc9hqz-1Xn-p2#comment-1567

### Other information:


## Jetpack product discussion



pc9hqz-1Xn-p2

## Does this pull request change what data or activity we track or use?



no

## Testing instructions:






* Setup Boost;
* Check the getting started page to make sure designs match the Pricing section.

Before:
![CleanShot 2023-08-17 at 15 56 35](https://github.com/Automattic/jetpack/assets/11799079/d316952a-8289-4ca8-a6c1-42caf2734c98)


After:
![CleanShot 2023-08-17 at 15 56 25](https://github.com/Automattic/jetpack/assets/11799079/f2c47878-f3db-42d2-b1e5-d367e86de0cf)
